crates/ruma-common/src/events/relation.rs

```diff
@@ -33,9 +33,14 @@ pub struct BundledAnnotation {
 
 #[cfg(feature = "unstable-msc2677")]
 impl BundledAnnotation {
-    /// Creates a new `BundledAnnotation` for a reaction.
+    /// Creates a new `BundledAnnotation` with the given key, type and count.
+    pub fn reaction(key: String, annotation_type: AnnotationType, count: UInt) -> Self {
```

Comment:
The type should probably come first (in the field list too?)

```suggestion
    pub fn new(annotation_type: AnnotationType, key: String, count: UInt) -> Self {
```